sphinxcontrib/openapi.py

```diff
@@ -138,6 +155,7 @@ class OpenApi(Directive):
     final_argument_whitespace = True        # path may contain whitespaces
     option_spec = {
         'encoding': directives.encoding,    # useful for non-ascii cases :)
+        'paths': directives.unchanged_required,
     }
 
```

Comment:
I think it'd be a good idea to do options transformations in `options` property, keeping the same interface it was before. Something like:

```python
@property
def options(self):
    options = super(OpenApi, self).options
    if 'endpoints' in options:
        options['endpoints'] = _clean_path_param(options['endpoints'])
    return options
```

and later pass it down:

```python
for line in openapi2httpdomain(spec, **self.options):
```

What do you think?